service-worker/README.md

```
@@ -0,0 +1,34 @@
+
+```
+  "scripts": {
+    "build": "tsc && vite build",
+    "build:uHTTP": "tsc && vite build && uhttp-post-build-react",
+  }
+```
+
+
+.env:
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)